### DISCUSSION TITLE
Modify attributes to the metrics globally

### DISCUSSION THREAD

**Discussion by shuowpro** [open-telemetry/opentelemetry-dotnet#5905] [Q&A]
For the trace, we are able to set tag for all the trace ever created by using the Processor. Is there something similar we can do for the metric? I want to add or delete some of the attribute for all the metrics, including the instrument metrics, but it looks like I cannot create a processor for the metric.

**Comment by cijothomas** [comment]
You are right. This capability do not exist for metrics today. There is no spec about a measurement processor, so this needs to first come via OTel spec.

(if you describe scenario more, maybe we can suggest workarounds/alternates)

**Reply by shuowpro** [reply]
Currently we are using azure monitors as well as geneva to collect the metrics, and we want to add some important attribute like pod name in k8s, but currently none of it support the resource attributes yet. So we are investigating some workaround. Is there any other way we can do to support that?

**Reply by cijothomas** [reply]
for things like pod name, Resource is the right way to model them.
If you report the issue with azure/Geneva, I can provide workarounds, if any, there.

Closing the issue here, as there is nothing actionable for this repo, given the user is affected backend specific issues.
